_posts/2021-06-08-How-to-invite-new-users-and-manage-existing-users-for-your-organization.md

```diff
@@ -0,0 +1,120 @@
+---
+date: 2021-06-08
+title: How to invite new users and manage existing users for your organization
+video_id: JC2yJnBXzLM
+description: You may want to invite new users to your organization, edit the roles of previous users or even assign an existing assistant to other users in the organization
+type:  
+sidebar:
+  - {id: step-1, text: Organization user}
+  - {id: step-2, text: Change the role of existing user}
+  - {id: step-3, text: Remove existing user }
+  - {id: step-4, text: Invite a new user}
+  - {id: step-5, text: Resending mail}
+  - {id: step-6, text: How to make assitant manager}
+  
+
+categories:
+  - How-To
+# set: getting-started
+# set_order: 3
+---
+
+
+You may want to invite new users to your organization, edit the roles of previous users or even assign an existing assistant to other users in the organization.
+
+These changes can be done by following the simple steps mentioned below.
+
+## Step 1
+
+Go to the Organization tab and click on 'Org users'.
+
+![image](../../images/Invite-1.png)
+
+
+## Step 2
+
+If you want to change the role of the existing user
+ 
+Click on 'action' and select 'make Org admin'.
+
+
+![image](../../images/Invite-2.png)
+
+Click on yes.
+
+![image](../../images/Invite-3.png)
+
+A new role will be assigned.
+
+## Step 3
+
+If you want to remove the existing user from Organization 
+
+Click on 'action' and select the user whom you want to assign leads that particular user was owning.
+
+![image](../../images/Invite-4.png)
+
+Click on proceed.
+
+![image](../../images/Invite-5.png)
+
+ The user will be removed from the organization and leads will be assigned to the selected user.
+
+
+## Step 4
+
+If you want to invite a new user to the organization.
+
+Click on 'invite', a pop-up will appear.
+
+![image](../../images/Invite-6.png)
+
+Fill in the details of the user you want to invite and click on 'invite user'.
+
+![image](../../images/Invite-7.png)
+
+After clicking on 'invite user', the registered email id of the new user will get an email id with a one-time password to log into.
+
+Note: Password needs to be changed within 24 hours.
+
+
+## Step 5
+
+There might be existing invited users that haven't logged in and changed the password within 24 hours.
```

Comment:
Invited users who have not logged in 24 hours of getting the one-time password would have to be reinvited. They cannot use forgot password functionality.